### DISCUSSION TITLE
How to monitor the progress of network picture loading?

### DISCUSSION THREAD

**Discussion by bingdu0** [coil-kt/coil#1136] [Q&A]
How to monitor the progress of network picture loading?

**Comment by colinrtwhite** [comment]
If you want to track downloading progress, you can add an [OkHttp interceptor](https://github.com/jobinlawrance/okhttp3-downloadprogress-interceptor) that monitors the download progress. You'll have to set a custom `ImageLoader` to set the `OkHttpClient` used for image requests.
